### ISSUE TITLE
Widget Support

### ISSUE THREAD

**Issue by dnet890** [lavalleeale/Lemmios#68]
Hi
It will be wonderful if we can view our home feeds directly without opening an application and widget is perfect for that. What do you think?

**Comment by lavalleeale** [comment]
Yes that feature already exists, the final widget the app provides shows posts from your subscribed communities

**Comment by dnet890** [comment]
Yes, But, it says not found
<img width="375" alt="19493629-6B47-4FE5-8210-392F181639CA" src="https://github.com/lavalleeale/Lemmios/assets/29157187/ede94911-1832-46e4-86f2-384eb8cf6071">

**Comment by lavalleeale** [comment]
Hmmm, can you hold down on one of the widget and send me a screenshot of what shows when you tap edit widget

**Comment by dnet890** [comment]
I tried to hold down the widget and there is no such thing called edit widget

![IMG_0043](https://github.com/lavalleeale/Lemmios/assets/29157187/718db6d4-137a-4c59-8659-f803ab415ead)
![IMG_0042](https://github.com/lavalleeale/Lemmios/assets/29157187/e10c9f74-4dd8-414f-a27c-764379d6ffe1)
![IMG_0041](https://github.com/lavalleeale/Lemmios/assets/29157187/97306977-28e3-49a8-a0fb-b817ab0eac8a)

**Comment by lavalleeale** [comment]
Ahh, is this the "latest post widget"? Try adding another widget and search "Lemmios" and the scroll all the way to the right for the "latest subscribed widget"

**Comment by dnet890** [comment]
Yes, it just shows blank like this
![IMG_0045](https://github.com/lavalleeale/Lemmios/assets/29157187/8468483d-8dfe-41e9-a172-6f2fc4221897)
![IMG_0044](https://github.com/lavalleeale/Lemmios/assets/29157187/e135b6ac-d429-4c90-a0fa-7c28f9c4aa48)
![IMG_0043-2](https://github.com/lavalleeale/Lemmios/assets/29157187/e895faed-3c4c-47d1-84f1-a5fe3a0d4738)
